indigo_app/static/javascript/indigo/views/annotations.js

```diff
@@ -541,28 +543,26 @@
         if (v.model === thread) {
           v.display();
 
-          this.gutter.activeItem = v;
+          v.el.active = true;
           // once it's rendered in the gutter, focus it
-          this.gutter.$nextTick(() => {
-            v.focus();
-            v.$el
-             .find('textarea')
-             .first()
-             .focus();
-          });
+
+          // this.gutter.$nextTick(() => {
```

Comment:
This functionality would be nice to preserve -- it means when you create a new comment the textarea is focused automatically.

Reply:
done